cmd/release/main.go

```diff
@@ -179,23 +190,36 @@ type config struct {
 }
 
 type runner struct {
+	builder  builder
 	releaser releaser
 	tagger   tagger
 	config   config
 }
 
+// TODO sha512 the artifacts!
+
 func (r runner) run() error {
-	fmt.Print("tagging repository ")
+	fmt.Printf("tagging repository %q ", r.config.version)
 	if err := r.tagger.tag(r.config.version); err != nil {
 		return err
 	}
 	fmt.Println("🐲")
-	fmt.Print("pushing tag ")
+	fmt.Printf("checking out tag %q ", r.config.version)
+	if err := r.tagger.checkout(r.config.version); err != nil {
+		return err
+	}
+	fmt.Println("🐲")
+	fmt.Printf("building artifacts %v ", r.config.artifacts)
```

Comment:
@vincepri this will print the list of artifacts that it expects to build. There still more to do here like checking for created files but I think that's out of scope for this PR